src/actinia_core/rest/resource_base.py

```diff
@@ -103,6 +103,7 @@ def __init__(self, resource_id=None, iteration=None, post_url=None):
         self.grass_base_dir = global_config.GRASS_GIS_BASE
         self.grass_start_script = global_config.GRASS_GIS_START_SCRIPT
         self.grass_addon_path = global_config.GRASS_ADDON_PATH
+        self.download_cache = global_config.DOWNLOAD_CACHE
```

Comment:
As far as I know, the download cache is always connected to the specific user, e.g. 
```
self.user_download_cache_path = os.path.join(self.config.DOWNLOAD_CACHE, self.user_id)
```
https://github.com/mundialis/actinia_core/blob/main/src/actinia_core/rest/download_cache_management.py#L126. This way the user can manipulate only the own cache.